llvm/test/CodeGen/SPIRV/memory_model_md.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s --check-prefix=SPV
+; RUN: llc -O0 -mtriple=spirv32-unknown-vulkan %s -o - | FileCheck %s --check-prefix=SPV
```

Comment:
why `vulkan`? `Physical32` is `Addresses`, hence kernel

Reply:
Mixing Physical addressing and shaders is fine as long as we are not targeting Vulkan.

Also `Simple` is deprecated. We might want to start phasing that out, but that is for another PR.